src/pybind/mgr/cephadm/serve.py

```diff
@@ -1696,7 +1696,12 @@ async def _run_cephadm(self,
                 self.log.debug('stdin: %s' % stdin)
 
             cmd = ssh.RemoteCommand(WHICH, ['python3'])
-            python = await self.mgr.ssh._check_execute_command(host, cmd, addr=addr)
+            try:
+                # when connection was broken/closed, retrying resets the connection 
```

Comment:
the line number jenkins is reporting is off from what this PR currently has, but I _think_ it's this comment and we'll need to fix this before this can get merged
```
cephadm/serve.py:1716:84: W291 trailing whitespace
1     W291 trailing whitespace
```